packages/import-bundle/test/test-compartment-wrapper.js

```diff
@@ -121,6 +121,8 @@ function check(t, c, odometer, n) {
     { message: /Not available/ },
     `${n} .constructor is tamed`,
   );
+
+  t.is(c.evaluate('WeakMap'), 'replaced');
```

Comment:
we should also test for `globalThis.WeakMap`, since that's the difference between `inescapableGlobalLexicals` and `inescapableGlobalProperties`